Python/symtable.c

```diff
@@ -260,7 +260,7 @@ symtable_new(void)
 #define COMPILER_STACK_FRAME_SCALE 3
 
 struct symtable *
-PySymtable_BuildObject(mod_ty mod, PyObject *filename, PyFutureFeatures *future)
+_PySymtable_Build(mod_ty mod, PyObject *filename, PyFutureFeatures *future)
```

Comment:
Why the rename?

Reply:
It's to prevent people to rely on it. It's to advertize: hey, it's a private function, don't use it!